.evergreen/preinstall.sh

```diff
@@ -13,9 +13,9 @@ echo "APPDATA: $APPDATA"
 echo "PATH: $PATH"
 
 # these are super useful if you want to run the smoke tests locally
-echo "DEV_VERSION_IDENTIFIER: $DEV_VERSION_IDENTIFIER"
-echo "EVERGREEN_BUCKET_NAME: $EVERGREEN_BUCKET_NAME"
-echo "EVERGREEN_BUCKET_KEY_PREFIX: $EVERGREEN_BUCKET_KEY_PREFIX"
+echo "export DEV_VERSION_IDENTIFIER=$DEV_VERSION_IDENTIFIER"
```

Comment:
This is to make it easy to copy/paste these into your local shell when working on the smoketests. Otherwise you have to transform the string manually every time.